hw/ip/sram_ctrl/dv/cov/sram_ctrl_unr_excl.el

```diff
@@ -3,149 +3,137 @@
 // Format Version: 2
-// Date: Tue Jun 20 19:02:19 2023
+// Date: Thu Dec 19 15:23:16 2024
 // ExclMode: default
 //==================================================
-CHECKSUM: "2736927696 2852341788"
-INSTANCE: tb.dut.u_tlul_adapter_sram.u_sram_byte.gen_integ_handling.u_sync_fifo.gen_normal_fifo.u_fifo_cnt
-ANNOTATION: "VC_COV_UNR"
-Block 12 "398307645" "wptr_o <= (wptr_o + {{(Width - 1) {1'b0}}, 1'b1});"
-ANNOTATION: "VC_COV_UNR"
-Block 21 "2517571270" "rptr_o <= (rptr_o + {{(Width - 1) {1'b0}}, 1'b1});"
-CHECKSUM: "4263908928 1069164873"
-INSTANCE: tb.dut.u_prim_ram_1p_scr
-ANNOTATION: "VC_COV_UNR"
-Block 23 "264295034" "rdata_o[k] = rdata[k];"
-CHECKSUM: "4224194069 2881513198"
+CHECKSUM: "3523621980"
+ANNOTATION: "[UNR] all inputs are constant"
+INSTANCE:tb.dut.u_seed_anchor.u_secure_anchor_buf.gen_generic.u_impl_generic
+CHECKSUM: "2099741489 2073313596"
+INSTANCE: tb.dut.u_reg_regs.u_status_bus_integ_error.wr_en_data_arb
+ANNOTATION: "[UNR] all inputs are constant"
+Block 2 "1620753216" "assign wr_data = d;"
+CHECKSUM: "2099741489 2073313596"
+INSTANCE: tb.dut.u_reg_regs.u_status_init_error.wr_en_data_arb
+ANNOTATION: "[UNR] all inputs are constant"
+Block 2 "1620753216" "assign wr_data = d;"
+CHECKSUM: "2099741489 2073313596"
+INSTANCE: tb.dut.u_reg_regs.u_status_escalated.wr_en_data_arb
+ANNOTATION: "[UNR] all inputs are constant"
+Block 2 "1620753216" "assign wr_data = d;"
+CHECKSUM: "998580748 2881513198"
 INSTANCE: tb.dut.u_tlul_lc_gate
 ANNOTATION: "VC_COV_UNR"
 Block 25 "1494489313" "block_cmd = 1'b1;"
 ANNOTATION: "VC_COV_UNR"
-Block 26 "2660541555" "state_d = StError;"
+Block 28 "1563566888" "state_d = StActive;"
 ANNOTATION: "VC_COV_UNR"
 Block 27 "4271836668" "if ((!flush_req_i))"
 ANNOTATION: "VC_COV_UNR"
-Block 28 "1563566888" "state_d = StActive;"
+Block 26 "2660541555" "state_d = StError;"
 ANNOTATION: "VC_COV_UNR"
 Block 36 "3828992405" "err_o = 1'b1;"
-CHECKSUM: "2574923469 1685788927"
-INSTANCE: tb.dut.u_tlul_adapter_sram.u_sram_byte
-ANNOTATION: "VC_COV_UNR"
-Block 25 "520728696" ";"
-CHECKSUM: "4224194069 4098474623"
+CHECKSUM: "998580748 4098474623"
 INSTANCE: tb.dut.u_tlul_lc_gate
 Fsm state_q "3443824386"
 ANNOTATION: "VC_COV_UNR"
 State StFlush "76"
-CHECKSUM: "74367784 3785313510"
-INSTANCE: tb.dut.u_reg_regs.u_reg_if
+Fsm state_q "3443824386"
 ANNOTATION: "VC_COV_UNR"
-Condition 18 "3340270436" "(addr_align_err | malformed_meta_err | tl_err | instr_error | intg_error) 1 -1" (5 "01000")
-CHECKSUM: "1335069400 4206106139"
+Transition StFlush->StActive "76->289"
+CHECKSUM: "704952876 1147758610"
+INSTANCE: tb.dut.u_prim_sync_reqack_data.u_prim_sync_reqack
+ANNOTATION: "[UNSUPPORTED] ACK can't come without REQ"
+Condition 1 "1414883863" "(src_req_i & src_ack_o) 1 -1"
+CHECKSUM: "998580748 639524789"
+INSTANCE: tb.dut.u_tlul_lc_gate
+ANNOTATION: "[LOWRISK] This happens in the 1st cycle after exiting reset. In order to cover it, need to drive TL items during reset, which isn't supported in the agent."
```

Comment:
An appropriate tag, given our employer :-)

I'm not completely sure about the explanation. Instead of "reset", do you mean something `lc_en_i`? I *think* this might be possible (but I'm willing to believe that we don't care!)

Reply:
This exclusion is already present [here](https://github.com/lowRISC/opentitan/blob/b46a8ab9e5dad3ad854e6ced434f4c39ccdfec3f/hw/ip/sram_ctrl/dv/cov/sram_ctrl_cov_excl.el#L108) in the existing sram_ctrL-cov_excl file.